create_scraper.py

```diff
@@ -61,16 +71,24 @@ def create_scraper_files(subs):
     init_target = os.path.join(target_dir, '__init__.py')
     populate_file(init_template, init_target, subs)
 
-    scraper_template = os.path.join(template_dir, 'region_scraper.txt')
-    scraper_target = os.path.join(target_dir, subs['region'] + '_scraper.py')
-    populate_file(scraper_template, scraper_target, subs)
+    if vendor == 'jailtracker':
+        scraper_template = os.path.join(template_dir, 'jailtracker', 'region_scraper.txt')
+    else:
+        scraper_template = os.path.join(template_dir, 'region_scraper.txt')
+    create_scraper(scraper_template)
 
-    yaml_template = os.path.join(template_dir, 'region.txt')
-    yaml_target = os.path.join(target_dir, subs['region'] + '.yaml')
-    populate_file(yaml_template, yaml_target, subs)
+    if vendor != 'jailtracker':
```

Comment:
I would change this to
```suggestion
    if not vendor:
```